OpenRA.Game/Graphics/VoxelRenderer.cs

```diff
@@ -315,7 +315,8 @@ public void EndFrame()
 				v.Second();
 			}
 
-			DisableFrameBuffer(fbo);
+			if (fbo != null)
+				DisableFrameBuffer(fbo);
```

Comment:
Are we sure this shouldn't still execute
```
			Game.Renderer.Flush();
			Game.Renderer.Device.DisableDepthBuffer();
```

I guess it never happens anyway, or we would have heard about a crash, but still...